src/WTG.Analyzers/Rules/Rules.xml

```diff
@@ -231,6 +231,11 @@
 			<description>Combining bool expressions in larger boolean expressions makes the code more complex than it needs to be.</description>
 			<tag>Unnecessary</tag>
 		</rule>
+		<rule id="16" name="InterpolatedStringMustBePurposeful" severity="Warning">
+			<title>Interpolated strings must be purposeful</title>
+			<message>Interpolated strings must be purposeful</message>
+			<description>Interpolated strings must be purposeful and include a combination of interpolation and interpolated string text</description>
```

Comment:
They don't need a combination, they just need more than one child.

`$"{a}{b}"` is valid.